router/core/access_log_field_handler_test.go

```diff
@@ -0,0 +1,117 @@
+package core
+
+import (
```

Comment:
I like that your wrote unit tests but we prefer integration tests. Please look in `router-tests`. We already have examples how to assert log statements.

The following cases needs to be covered:

- Add a custom router log field that reacts to an error
- Add a custom subgraph log field that reacts to an error
- Ensure that invalid expression are catched.

Reply:
Tests added